translation/en-US.ts

```diff
@@ -177,6 +179,30 @@ export default {
   range_file: 'file',
   range_module: 'module',
 
+  //finance Page
+  bill_id: 'bill id',
+  bill_createAt: 'createAt',
+  bill_location: 'location',
+  bill_createBy: 'createBy',
+  bill_type: 'type',
+  bill_price: 'price',
+  bill_invoice: 'invoice',
+  bill_remark: 'remark',
+  bill_travelFundTask: 'travelFundTask',
```

Comment:
```suggestion
  bill_travelFundTask: '?',
```
”差旅志愿任务“ 中文指的是啥？
这个英文表达不 make sense

Reply:
@luojiyin1987 
我看了水歌给的链接里面的对应选项，
那用 Volunteer Task(s) Assigned （因为可以多选）
就是被分配的志愿工作